### PR TITLE
Update repositories-recommendations.md

### PR DESCRIPTION
**Pull request by lhoestq**

### CODE REVIEW

File: docs/hub/repositories-recommendations.md [hunk 1]
```diff
@@ -59,7 +59,9 @@ adding around 50-100 files per commit.
 
 ## Sharing large datasets on the Hub
 
-One key way in which Hugging Face supports the machine learning ecosystem is by hosting datasets on the Hub, including very large datasets. To ensure we can effectively support the open-source ecosystem, we ask that you let us know in advance via datasets at huggingface.co or on [our Discord](http://hf.co/join/discord) if you are uploading datasets above a couple of hundred GBs or TBs of data.
+One key way in which Hugging Face supports the machine learning ecosystem is by hosting datasets on the Hub, including very large datasets. However if your dataset is bigger than 300GBs you will need to ask us to grant more storage.
```
Comment (osanseviero) [inline docs/hub/repositories-recommendations.md:62]:
```suggestion
One key way Hugging Face supports the machine learning ecosystem is by hosting datasets on the Hub, including very large ones. However, if your dataset is bigger than 300GBs, you will need to ask us to grant more storage.
```

File: docs/hub/repositories-recommendations.md [hunk 2]
```diff
@@ -59,7 +59,9 @@ adding around 50-100 files per commit.
...
+
+In this case and to ensure we can effectively support the open-source ecosystem, we require you to let us know via datasets at huggingface.co.
```
Comment (osanseviero) [inline docs/hub/repositories-recommendations.md:64]:
```suggestion
In this case, to ensure we can effectively support the open-source ecosystem, we require you to let us know via datasets at huggingface.co.
```

**Review by osanseviero — APPROVED** [review]

File: docs/hub/repositories-recommendations.md [hunk 3]
```diff
@@ -59,7 +59,9 @@ adding around 50-100 files per commit.
 
 ## Sharing large datasets on the Hub
 
-One key way in which Hugging Face supports the machine learning ecosystem is by hosting datasets on the Hub, including very large datasets. To ensure we can effectively support the open-source ecosystem, we ask that you let us know in advance via datasets at huggingface.co or on [our Discord](http://hf.co/join/discord) if you are uploading datasets above a couple of hundred GBs or TBs of data.
+One key way Hugging Face supports the machine learning ecosystem is by hosting datasets on the Hub, including very large ones. However, if your dataset is bigger than 300GBs, you will need to ask us to grant more storage.
```
Comment (julien-c) [inline docs/hub/repositories-recommendations.md:62]:
```suggestion
One key way Hugging Face supports the machine learning ecosystem is by hosting datasets on the Hub, including very large ones. However, if your dataset is bigger than 300GB, you will need to ask us to grant more storage.
```

File: docs/hub/repositories-recommendations.md [hunk 4]
```diff
@@ -59,7 +59,9 @@ adding around 50-100 files per commit.
...
+
+In this case, to ensure we can effectively support the open-source ecosystem, we require you to let us know via datasets at huggingface.co.
```
Comment (julien-c) [inline docs/hub/repositories-recommendations.md:64]:
```suggestion
In this case, to ensure we can effectively support the open-source ecosystem, we require you to let us know via datasets@huggingface.co.
```

be consitent with the paragraph above

**Pull request merged**